### PR TITLE
DAT-3164 move insights icon css to mixin and reuse it in rail module and speci…

### PR DESCRIPTION
**Pull request by RafLeszczynski**
…al page css

Jira ticket:
https://wikia-inc.atlassian.net/browse/DAT-3164

### CODE REVIEW

**Comment by RafLeszczynski** [comment]
@kamilkoterba @lukaszkonieczny @adamkarminski could you please take a look?

**Comment by lukaszkonieczny** [comment]
LGTM

File: extensions/wikia/Insights/styles/_insights-icons-mixin.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@mixin insigths-icons-dark {
```
Comment (sebastianmarzjan) [inline extensions/wikia/Insights/styles/_insights-icons-mixin.scss:1]: Typo ( s/insigths/insights )

File: extensions/wikia/Insights/styles/_insights-icons-mixin.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	.insights-icon-wantedpages {
+		background-image: url('/extensions/wikia/Insights/images/wantedpages_dark.svg'); /* inline */
+	}
+
+	.insights-icon-nonportableinfoboxes {
+		background-image: url('/extensions/wikia/Insights/images/genericinsights_dark.svg'); /* inline */
+	}
+}
+
+@mixin insigths-icons-light {
```
Comment (sebastianmarzjan) [inline extensions/wikia/Insights/styles/_insights-icons-mixin.scss:27]: Same typo here

**Comment by sebastianmarzjan** [comment]
Thanks for the extraction to mixins!
Please fix the typos though to allow for easier grepping and less confusion in future

**Comment by RafLeszczynski** [comment]
@SebastianMarzjan done, waiting for CR https://github.com/Wikia/app/pull/8378

**Pull request merged**